prac_08/unreliable_car.py

```diff
@@ -0,0 +1,17 @@
+from prac_08.car import Car
+import random
+
+
+class UnreliableCar(Car):
+
+    def __init__(self, name, fuel, reliability):
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        if random.randint(0, 100) < self.reliability:
+            super().drive(distance)
+        else:
+            distance = 0
+        return distance
```

Comment:
@4ndrewJ - we don't want to do this because it's possible the `super().drive` method does something else that we need/want. Better would be something like the solutions, or you could make it shorter with:
```
    if random.randint(0, 100) < self.reliability:
        distance = 0
    return super().drive(distance)
```